### PR TITLE
feat(content-item): add content item template - INNO-1498

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

specs available [on the wiki](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Content+item+-+Design+specifications)

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 60d4e6a4daf838bcb8cf4ab95cd2af2f640ce72e

https://deploy-preview-1150--europa-component-library.netlify.com

File: src/website/src/pages/ec/templates/compositions/content-item/docs/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  description="Do not mix different content items on the same list as it will affect scannability."
+/>
+
+## When to use
+
+- use content items when you want to display a list of events, news, agenda or other similar types of data
+
+## When not to use
+
+- do not use for navigation purpose; use <Link to="/ec/templates/compositions/navigation-lists/usage/">navigation lists</Link> instead
```
Comment (yhuard) [inline src/website/src/pages/ec/templates/compositions/content-item/docs/usage.mdx:70]: you forgot to import `Link` (that's why it doesn't work)

File: src/systems/ec/implementations/react/templates/pages/campaign/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "@ecl/ec-react-campaign-page",
-  "author": "ecropean Commission",
```
Comment (yhuard) [inline src/systems/ec/implementations/react/templates/pages/campaign/package.json:4]: 🤣

File: src/systems/ec/implementations/react/templates/pages/campaign/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+  "author": "european Commission",
```
Comment (yhuard) [inline src/systems/ec/implementations/react/templates/pages/campaign/package.json]:
```suggestion
  "author": "European Commission",
```

File: src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Icon from '@ecl/ec-react-component-icon';
+import Link from '@ecl/ec-react-component-link';
+import { UnorderedList, UnorderedListItem } from '@ecl/ec-react-component-list';
+
+export default () => (
+  <article className="ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m">
+    <div className="ecl-u-flex-grow-1">
+      <div className="ecl-u-type-s ecl-u-type-color-grey-75">
+        NEWS ARTICLE | 17 October 2019
```
Comment (yhuard) [inline src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx]: can you use the `<time>` tag here? :)

File: src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          <span className="ecl-u-type-s ecl-u-ml-s">Brussels, Belgium</span>
+        </UnorderedListItem>
+        <UnorderedListItem className="ecl-u-d-flex ecl-u-align-items-center ecl-u-type-color-grey-75">
+          <Icon shape="general--livestreaming" size="m" />
+          <span className="ecl-u-type-s ecl-u-ml-s">Live stream available</span>
+        </UnorderedListItem>
+      </UnorderedList>
+    </div>
+
+    <div
```
Comment (yhuard) [inline src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx:33]: can we find another solution do display the image in a more semantically-correct way?

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx (hunk 5 above)
Comment (emeryro) [reply src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx:33]: well, the problem is that we have to fit the image in a predifined size/ratio. So we can't use a simple `` or the image will be deformed.
That's why I use the background-image.
I don't really see other ways to do it...

File: src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Icon from '@ecl/ec-react-component-icon';
+import Link from '@ecl/ec-react-component-link';
+import { UnorderedList, UnorderedListItem } from '@ecl/ec-react-component-list';
+
+export default () => (
+  <article className="ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m">
+    <div className="ecl-u-flex-grow-1">
+      <div className="ecl-u-type-s ecl-u-type-color-grey-75">
+        NEWS ARTICLE | <time>17 October 2019</time>
```
Comment (yhuard) [inline src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx]: `datetime` attribute is missing

File: src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx (hunk 5 above)
Comment (yhuard) [reply src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx:33]: at least, could you use `role="img"` and an `aria-label` attribute? :)

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/implementations/react/templates/compositions/content-item/package.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
+{
+  "private": true,
+  "name": "@ecl/ec-react-composition-content-item",
+  "author": "european Commission",
```
Comment (yhuard) [inline src/systems/ec/implementations/react/templates/compositions/content-item/package.json]:
```suggestion
  "author": "European Commission",
```

File: src/systems/eu/implementations/react/templates/compositions/content-item/package.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
+{
+  "private": true,
+  "name": "@ecl/eu-react-composition-content-item",
+  "author": "european Commission",
```
Comment (yhuard) [inline src/systems/eu/implementations/react/templates/compositions/content-item/package.json]:
```suggestion
  "author": "European Commission",
```

**Review by yhuard — CHANGES_REQUESTED** [review]

**Review by yhuard — APPROVED** [review]

File: src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+        </UnorderedListItem>
+      </UnorderedList>
+    </div>
+
+    <div
+      role="img"
+      aria-label="Example image"
+      className="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-ml-xl ecl-u-d-lg-none"
+      style={{
+        backgroundImage: 'url("/example-image.jpg")',
```
Comment (yhuard) [inline src/systems/ec/implementations/react/templates/compositions/content-item/examples/Default.jsx]: Please use the images from our S3 bucket:
```suggestion
        backgroundImage: 'url("https://inno-ecl.s3.amazonaws.com/media/examples/example-image.jpg")',
```

**Review by yhuard — CHANGES_REQUESTED** [review]

**Pull request merged**